app/src/main/res/layout/activity_main.xml

```diff
@@ -36,9 +36,24 @@
             android:padding="@dimen/grid_1"
             android:text="@string/dialogPopUpBtn"
             android:textSize="@dimen/text_1" />
+        <TextView
+            android:id="@+id/colorPickerBtn"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="@drawable/text_btn_background_"
+            android:padding="@dimen/grid_1"
+            android:text="@string/dialogPopUpBtn"
+            android:textSize="@dimen/text_1" />
 
 
```

Comment:
extra line, I'll only put this once so you can sweep these when you are at the point where that polish is meaningful